customer-service/src/main/kotlin/cz/kss/proj/clientservice/dto/customer-contract.kt

```diff
@@ -0,0 +1,67 @@
+package cz.kss.proj.clientservice.dto
```

Comment:
I personally prefer camel case, but it's a matter of agreement. I'd rename the file from `customer-contract` to `CustomerContract`.

Reply:
Maybe I'd also move up some of the classes into standalone files.

Reply:
OK, I will rename it

Reply:
I fix it